Sources/MapboxMaps/Annotations/AnnotationManager.swift

```diff
@@ -93,7 +93,11 @@ public class AnnotationManager {
     /**
      Options used by the annotation manager.
      */
-    internal var annotationOptions: AnnotationOptions
+    public var options = AnnotationOptions() {
```

Comment:
Can we just make this a let constant, rather than have a log warning?

Reply:
This is keeping the behavior that existed earlier. In any case, this entire file will be going away soon with the annotations refactor.

Reply:
Sounds good